src/menu/menu.c

```diff
@@ -893,6 +895,161 @@ menu_hide_submenu(struct server *server, const char *id)
 	}
 }
 
+/*
+ * This is client-send-to-menu
+ * an internal menu similar to root-menu and client-menu
+ *
+ * This will look at workspaces and produce a menu
+ * with the workspace names that can be used with
+ * SendToDesktop, left/right options are included.
+ */
+void
+create_client_send_to_menu(struct server *server)
+{
+	struct menu *menu = menu_get_by_id(server,
+			"client-send-to-menu");
+
+	if (menu) {
+		struct menuitem *item, *next;
+		wl_list_for_each_safe(item, next, &menu->menuitems, link) {
+			item_destroy(item);
+		}
+	} else {
+		menu = menu_create(server, "client-send-to-menu", "Send to...");
+	}
+
+	menu->label = xstrdup("Send to...");
+	menu->size.height = 0;
+
+	if (wl_list_empty(&menu->menuitems)) {
+		struct workspace *workspace;
+
+		wl_list_init(&menu->menuitems);
+		wl_list_for_each(workspace, &server->workspaces, link) {
+			if (!strcmp(workspace->name,
+						server->workspace_current->name)) {
+				current_item = item_create(menu, strdup_printf(">%s<",
+							workspace->name), false);
+			} else {
+				current_item = item_create(menu, workspace->name, false);
```

Comment:
Please add an inline comment like `item_create(menu, workspace->name, /*show_arrow*/ false);`, same for the other `item_create()` calls.